readme.md

```
@@ -1,5 +1,204 @@
+```
+
+### 3. カード画像の取得
+```bash
+cd tools/scraping
+pip install -r requirements.txt
+python scrape_images.py
+```
+
+## 📊 データベース構造
+
+### メインテーブル
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)